src/constants.js

```diff
@@ -1,9 +1,12 @@
 /* eslint-disable import/prefer-default-export */
 export const ExamStatus = Object.freeze({
```

Comment:
@ihor-romaniuk could you please clarify who is responsible to add pages for the next Exam statuses:
download_software_clicked
ready_to_start
declined -- not sure it is required separate page
error
ready_to_resume -- ditto, need to be checked is separate page required.

Reply:
We didn't discuss other instructions. I guess that we need to create corresponding tickets for creating them.

Reply:
Got it will do!